airflow/providers/databricks/hooks/databricks.py

```diff
@@ -519,13 +519,26 @@ def delete_run(self, run_id: int) -> None:
         json = {"run_id": run_id}
         self._do_api_call(DELETE_RUN_ENDPOINT, json)
 
-    def repair_run(self, json: dict) -> None:
+    def repair_run(self, json: dict) -> int:
         """
         Re-run one or more tasks.
 
         :param json: repair a job run.
         """
-        self._do_api_call(REPAIR_RUN_ENDPOINT, json)
+        response = self._do_api_call(REPAIR_RUN_ENDPOINT, json)
+        return response["repair_id"]
+
+    def get_latest_repair_id(self, run_id: int) -> Optional[int]:
+        """
+        get latest repair id if any exist for run_id else None
+        """
```

Comment:
```suggestion
        """Get latest repair id if any exist for run_id else None."""
```

This should fix another Static Code Check

```console
airflow/providers/databricks/hooks/databricks.py:532:9: D200 One-line docstring should fit on one line
airflow/providers/databricks/hooks/databricks.py:532:9: D400 First line should end with a period
```